src/InvokePlaster.ps1

```diff
@@ -1,47 +1,58 @@
-<#
-NOTE TO DEVELOPERS:
-All text displayed to the user except for Write-Debug (or $PSCmdlet.WriteDebug()) text must be added to the
-string tables in:
-    en-US\Plaster.psd1
-    Plaster.psm1
-
-If a new manifest element is added, it must be added to the Schema\PlasterManifest-v1.xsd file and then
-processed in the appropriate function in this script.  Any changes to <parameter> attributes must be
-processed not only in the ProcessParameter function but also in the dynamicparam function.
-
-Please follow the scripting style of this file when adding new script.
-#>
+## DEVELOPERS NOTES & CONVENTIONS
+##
+##  1. All text displayed to the user except for Write-Debug (or $PSCmdlet.WriteDebug()) text must be added to the
+##     string tables in:
+##         en-US\Plaster.psd1
+##         Plaster.psm1
+##  2. If a new manifest element is added, it must be added to the Schema\PlasterManifest-v1.xsd file and then
+##     processed in the appropriate function in this script.  Any changes to <parameter> attributes must be
+##     processed not only in the ProcessParameter function but also in the dynamicparam function.
+##
+##  3. Non-exported functions (subroutines) should avoid using PowerShell standart Verb-Noun naming convention.
```

Comment:
I'll fix it.  That's what I get for copy/pasting comments - https://github.com/PowerShell/platyPS/blob/master/src/platyPS/platyPS.psm1#L6  :-)